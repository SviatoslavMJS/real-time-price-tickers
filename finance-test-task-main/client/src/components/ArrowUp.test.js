import renderer from 'react-test-renderer';
import ArrowUp from './ArrowUp';

test('Testing ArrowUp', () => {
    const component = renderer.create(
        <ArrowUp />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
