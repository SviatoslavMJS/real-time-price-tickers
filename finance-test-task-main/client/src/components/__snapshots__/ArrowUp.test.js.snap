// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing ArrowUp 1`] = `
<svg
  className="svg_up"
>
  <path
    d="M4 12l1.41 1.41L11 7.83V20h2V7.83l5.58 5.59L20 12l-8-8-8 8z"
  />
</svg>
`;
